### DISCUSSION TITLE
Error in proxy setup - self signed in chain.

### DISCUSSION THREAD

**Discussion by aasmundkamphaug** [Security-Onion-Solutions/securityonion#11783] [2.4] [answered]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

16

### Storage for /

1 T

### Storage for /nsm

600

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When i try to configure proxy for the internett access, i get an error that the proxy has selfsigned in a part of the chain. Is there a way to skip verification ?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
This may require custom configuration for each component that you're trying to connect through the proxy. 

If all else fails, you can always do perform an airgap installation and it won't need your proxy at all.

**Reply by aasmundkamphaug** [reply]
Thx for the answer

**Reply by aasmundkamphaug** [reply]
@dougburks one follow up question. When using a proxy setup; is it so that only the manager needs access to the internett/proxy or do all nodes need it
